src/client/theme-default/components/VPFlyout.vue

```diff
@@ -90,7 +90,7 @@ function onBlur() {
 .button {
   display: flex;
   align-items: center;
-  padding: 0 12px;
+  padding: 4px 12px;
```

Comment:
When text wraps, it's useful to have a bit of vertical padding acting as a security to ensure that the text will not end up touching the top or bottom edge of the navbar.

With the specified line-height of 20px:

- Height for two lines `4 + 2x20 + 4 = 48`
- Height for three lines: `4 + 3x20 + 4 = 68`

We could theoretically use 8px of vertical padding and exactly match the default `--vp-nav-height-mobile` value (56px) for 2 lines of text. But that could get in the way of users making a custom theme and trying to use a lower `--vp-nav-height-mobile` value.